### PR TITLE
Add Bitly URL Shortener Extension

### PR DESCRIPTION
**Pull request by blessanm86**
## Description

A quick command to shorten a url in the clipboard with Bitly.

## Type of change

- New extension

## Screencast
Screenshot 2021-12-01 at 2 24 28 PM

Screenshot 2021-12-01 at 2 15 14 PM

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I checked that extension's icon has size of 512x512

### CODE REVIEW

File: extensions/bitly-url-shortener/package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "bitly-url-shortener",
+  "title": "Bitly URL Shortener",
+  "description": "Quickly shorten your url’s with Bitly",
```
Comment (fe9lix) [inline extensions/bitly-url-shortener/package.json:5]:
```suggestion
  "description": "Quickly shorten the current clipboard URL with Bitly",
```

File: extensions/bitly-url-shortener/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      }),
+    });
+
+    const { errors, link } = (await response.json()) as { link: string; errors?: [] };
+    if (errors) {
+      throw new Error("Invalid URL String");
+    }
+
+    await copyTextToClipboard(link);
+    await showHUD("Copied shorten url to clipboard");
```
Comment (fe9lix) [inline extensions/bitly-url-shortener/src/index.tsx:31]:
```suggestion
    await showHUD("Copied shortened URL to clipboard");
```

**Review by fe9lix — CHANGES_REQUESTED** [review]
Hey @blessanm86! Thanks for the contribution and sorry for the review delay. I've left some smaller inline comments.

**Comment by blessanm86** [comment]
@fe9lix Thanks, I have updated the PR.

**Review by fe9lix — APPROVED** [review]

**Pull request merged**